infra/docker/web/Dockerfile

```diff
@@ -0,0 +1,101 @@
+FROM php:7.4-fpm-alpine
+
+EXPOSE 80
+
+# Create some environment variables if not present.
+# TBD - We're going to be injecting a lot of values via GitHub Actions.
+
+# Configure timezone
+RUN ln -s /usr/share/zoneinfo/Etc/UTC /etc/localtime
+
+# Install Libraries
+RUN apk add --update --no-cache \
+    libgd \
+    imagemagick \
+    html2text \
+    nginx \
+    freetype \
+    postgresql \
+    tidyhtml
+
+# Install PHP Extensions not already present in alpine release
+# RUN docker-php-ext-install -j$(nproc) \
+#     opcache \
+#     pgsql \
+#     tidy \
+#     gd
+
+# Memcached PHP lib
+RUN set -xe && \
+    cd /tmp/ && \
+    apk add --no-cache --update --virtual .phpize-deps $PHPIZE_DEPS && \
```

Comment:
Where does `$PHPIZE_DEPS` come from?

Reply:
```
/ # echo $PHPIZE_DEPS
autoconf dpkg-dev dpkg file g++ gcc libc-dev make pkgconf re2c
```

Now if you're asking me where it gets that variable, great question! Let me know if you find out.

Reply:
Oh, seems to be from the `php` container: https://github.com/docker-library/php/blob/master/Dockerfile-alpine.template#L6